src/actions/registerUserActions.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import { REGISTER_USER_SUCCESS, REGISTER_USER_ERROR } from './types';
+
+export const fetchUsers = (postData) => dispatch => {
+  axios
+    .post('http://127.0.0.1:8000/api/users/', postData)
+    .then((response) => {
+      localStorage.setItem('auth_token', response.data.user.auth_token);
+      dispatch({ type: REGISTER_USER_SUCCESS, payload: true });
+      toast.success('Signup successful', { autoClose: false });
```

Comment:
Good work @MuhanguziDavid! I am impressed with your test coverage!
You should consider changing the `autoClose` functionality.
Something of the sort `{ autoClose: 3500, hideProgressBar: true }` the latter part removes the default progress bar. The same toast change applies to the `error` toast on line 33.

Reply:
The autoClose functionality has been changed